addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -800,7 +800,7 @@ var SnippetsMenu = Widget.extend({
                 var parentDisplay = parentCss.display;
                 var parentFlex = parentCss.flexDirection;
 
-                if ($zone.prev('.oe_drop_zone:visible').length === 0) {
+                if ($zone.prev('.oe_drop_zone:visible').length === 0 && $zone.prev('.oe_drop_clone').prev('.oe_drop_zone:visible').length === 0) {
```

Comment:
I think the code would be easier to read and more consitent by:
- Adding automatically a drop zone after the clone (whatever the condition, you are always allow to redrop the snippet at the same location)
- Changing the condition here in a more simple `!$zone.prev('.oe_drop_zone:visible, .oe_drop_clone').length`

Reply:
@qsm-odoo ok done :+1: